pages/api/revalidate.js

```diff
@@ -22,6 +23,12 @@ async function handler(req, res) {
     return res.status(500).json({ error: `The parameter "uri" is required.` });
   }
 
+  try {
```

Comment:
since we don't have an app directory setup, squash the error message that will throw when trying to revalidate path